_data/people.yml

```diff
@@ -142,6 +142,20 @@
       icon: linkedin
       name: Linkedin
 
+- name: Oleksandr Yardas
+  pic: /img/people/yardaso.jpeg
+  dates: 2020-present
+  role: Graduate Research Assistant
+  description: 'Olek is a Masters student interested in passive safety and operations, microreactors, and decentralized power generation. He is still trying to figure out how to integrate these interests into a viable career.'
+  social:
+      - url: mailto:oyardas2@illinois.edu
+        icon: envelope-o
+        name: email
+      - url: https://github.com/yardasol
+        icon: github-alt
+        name: Github
+
```

Comment:
```suggestion
```

Reply:
I assume this is because there is nothing notable/relevant to arfc on my github right now? The change has been made.